doc/newsboat.asciidoc

```diff
@@ -1022,7 +1022,10 @@ by " ") and "download error" (indicated by "x").
 Identifier:Meaning
 [[articlelist-format-a]]<<articlelist-format-a,+a+>>:Article author
 [[articlelist-format-D]]<<articlelist-format-D,+D+>>:Publication date. This can be tweaked further with <<datetime-format,+datetime-format+>>
-[[articlelist-format-f]]<<articlelist-format-f,+f+>>:Article flags
+[[articlelist-format-f]]<<articlelist-format-f,+f+>>:Combination of unread/deleted fields and article flags
```

Comment:
The doc is not entirely correct here about the article flags: they're replaced by an exclamation mark or a space (see `gen_flags()`). I'd change this to:

> Two characters: 1) "N" if article is unread, "D" if article is deleted, a space otherwise; 2) "!" if article has flags, a space otherwise.

Reply:
Good suggestion. Thanks!